conformance/utils/kubernetes/helpers.go

```diff
@@ -203,6 +203,12 @@ func NamespacesMustBeAccepted(t *testing.T, c client.Client, timeoutConfig confi
 					t.Logf("%s/%s Gateway not ready yet", ns, gw.Name)
```

Comment:
This might be slightly more clear:
```suggestion
					t.Logf("%s/%s Gateway not accepted yet", ns, gw.Name)
```

Reply:
Updated, thanks!